backend/src/v5/models/tickets.comments.js

```diff
@@ -82,12 +96,28 @@ TicketComments.updateComment = async (teamspace, oldComment, updateData) => {
 	};
 
 	await updateOne(teamspace, { _id: oldComment._id }, updateObj);
+
+	publish(events.UPDATE_COMMENT, {
+		teamspace,
+		project,
+		model,
+		data: {
+			ticket,
+			_id: oldComment._id,
+			message: updateData.message,
+			images: updateData.images,
+			author: oldComment.author,
+			updatedAt: formattedComment.updatedAt,
+		},
+	});
 };
 
-TicketComments.deleteComment = async (teamspace, oldComment) => {
+TicketComments.deleteComment = async (teamspace, project, model, ticket, oldComment) => {
 	const formattedComment = getUpdatedComment(oldComment, { deleted: true });
 	await updateOne(teamspace, { _id: oldComment._id },
 		{ $set: { ...formattedComment }, $unset: { message: 1, images: 1 } });
+
+	publish(events.UPDATE_COMMENT, { teamspace, project, model, data: { ticket, _id: oldComment._id, deleted: true } });
```

Comment:
I think we should provide `updatedAt` here as well so the UI knows when it was deleted (I might've missed it on the schema)